### PR TITLE
Add a few things to rewrite

### PR DESCRIPTION
**Pull request by hyknn**
What's added:
- Laravel Sail (Development purpose)
- Add demo user seed
- Add client home page (empty page with navigation bar)

### CODE REVIEW

File: src/resources/views/components/fossbilling-logo.blade.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (BelleNottelling) [inline src/resources/views/components/fossbilling-logo.blade.php:1]: This causes Laravel to error out when trying to view the index page

Comment (hyknn) [reply src/resources/views/components/fossbilling-logo.blade.php:1]: Ahh, really? I'm using application-logo.blade.php as reference tho, and it works just fine on my local

image

Comment (evrifaessa) [reply src/resources/views/components/fossbilling-logo.blade.php:1]: @BelleNottelling `php artisan key:generate` does the trick.
It works for me. However, we should document the installation process.

image

Comment (hyknn) [reply src/resources/views/components/fossbilling-logo.blade.php:1]: > @BelleNottelling `php artisan key:generate` does the trick. It works for me. However, we should document the installation process.
> 
> image

1. Clone / download the repo
2. run "composer install" for installing required package from composer
3. run "npm install" for installing required package from npm
4. copy and rename .env.example to .env
5. set .env file according your environment
6. run "php artisan key:generate" for generate Laravel app key
7. run "php artisan migrate:fresh" for migrate database tables

Anything I am missing?

Comment (jaapmarcus) [reply src/resources/views/components/fossbilling-logo.blade.php:1]: I assume we also not to run gulp to build the artifacts

Comment (BelleNottelling) [reply src/resources/views/components/fossbilling-logo.blade.php:1]: > 

Ah thanks, it didn't cross my mind

**Comment by evrifaessa** [comment]
@BelleNottelling anything blocking this from being merged?

**Comment by BelleNottelling** [comment]
> @BelleNottelling anything blocking this from being merged?

Not that I can think of

**Pull request merged**